core/udp_connection.h

```diff
@@ -30,9 +30,6 @@ class UdpConnection : public Connection
 
     static void receive(UdpConnection *parent);
 
-    // This port is shared with mavros, so either one, this SDK or mavros can be used.
```

Comment:
I was going to complain that we lose this comment but I see you moved it.

Reply:
Yeah, I tend to be aggressive with comments. Good that you keep an eye on them xD.